pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,53 @@
+class Push(object):
+    """This represents a Push"""
+
+    def __init__(self, **kwargs):
+        """
+            We set Push object default as dict, and we will
+            give each push a push id when we init it.
+        """
+        self._id, self._push = kwargs.popitem()
+
+    def __repr__(self):
+        return "<Push id:%s info:%s>" % (self._id, self._push)
+
+    def __cmp__(self, other):
+        if self.id and other.id:
+            return cmp(self.id, other.id)
```

Comment:
:+1: good!

Why do you add the if clause?
Could you please write few tests testing this functionality?